python/pyspark/ml/feature.py

```diff
@@ -3852,9 +3852,21 @@ def from_arrays_of_labels(cls, arrayOfLabels, inputCols, outputCols=None,
     def labels(self):
         """
         Ordered list of labels, corresponding to indices to be assigned.
+
+        .. deprecated:: 3.1.0
+            It will be removed in future versions. Use `labelsArray` method instead.
         """
         return self._call_java("labels")
 
+    @property
+    @since("3.1.0")
```

Comment:
As this is backport to 3.0.2 #30580, we need to change this to 3.0.2.

Reply:
This is pretty minor. Do we need a follow-up PR? @HyukjinKwon

Reply:
Yeah, let's change.

Reply:
okay, will create one later.